libraries/disp3D/engine/view/view3D.h

```diff
@@ -51,6 +51,7 @@
 #include <QPointer>
 #include <QObjectPicker>
 #include <Qt3DCore>
+#include "orbitalcameracontroller.h"
```

Comment:
move `#include "orbitalcameracontroller.h"` to line 44 right after `#include "../../disp3D_global.h"`